x-pack/plugins/reporting/public/management/components/report_info_flyout_content.tsx

```diff
@@ -33,14 +33,15 @@ const UNKNOWN = i18n.translate('xpack.reporting.listing.infoPanel.unknownLabel',
 
 interface Props {
   info: Job;
+  isServerless: boolean;
```

Comment:
2. Let's have this expect the `ClientConfigType` object
```suggestion
  config: ClientConfigType;
```